spec/defaultHtmlStepUi.spec.js

```diff
@@ -6,7 +6,7 @@ global.document = DOM.window.document;
 describe('Sequencer step HTML', function() {
 
   var DefaultHtmlStepUi = require('../examples/lib/defaultHtmlStepUi');
-  var sequencer = require('../src/ImageSequencer')();
+  var sequencer = require('../src/ImageSequencer')({useWasm:true});
```

Comment:
Is this `useWasm: true` necessary since it's a default?

Reply:
Actually I added them in the starting. But no need now so deleting them.